src/lifx/validate.js

```diff
@@ -82,8 +82,8 @@ validate.irBrightness = function(brightness, context) {
  * @param {String} context validation context
  */
 validate.optionalKelvin = function(kelvin, context) {
-  const message = 'LIFX %s expects kelvin to be a number between 2500 and 9000';
   if (kelvin !== undefined) {
+    const message = `LIFX %s expects kelvin to be a number between ${constants.HSBK_MINIMUM_KELVIN} and ${constants.HSBK_MAXIMUM_KELVIN}`;
```

Comment:
I agree with this change, but also, the min and max are different for different bulbs so technically these aren't constants at all. These are constants from the min and max across their entire line of bulbs, but then again, my Day & Night bulbs go down to `1500K` and only up to `4000K`.

Instead of constants, can we pull these values from the information on that specific bulb?

Reply:
I also so that during the implementation.
I will prepare another pr for this is in the code base.

Reply:
Cool. I'll resolve this then.